packages/crayons-core/src/components/inline-message/inline-message.tsx

```diff
@@ -0,0 +1,157 @@
+import {
+  Component,
+  Prop,
+  h,
+  Host,
+  Watch,
+  Element,
+  Method,
+  EventEmitter,
+  Event,
+} from '@stencil/core';
+
+const iconMap = {
+  error: 'alert',
+  warning: 'info',
+  info: 'info',
+  success: 'success',
+};
+
+const iconColorMap = {
+  error: '#e43538',
+  warning: '#c7502f',
+  info: '#264966',
+  success: '#00795b',
+};
+
+@Component({
+  tag: 'fw-inline-message',
+  styleUrl: 'inline-message.scss',
+  shadow: true,
+})
+export class InlineMessage {
+  @Element() host!: HTMLElement;
+
+  private autoHideTimeout;
+
+  /**
+   * Makes the inline message closable.
+   */
+  @Prop() closable = false;
```

Comment:
Lets keep this as default true!

Reply:
sure, made this and open true by default